src/domain/schemas/document-request-schemas.js

```diff
@@ -0,0 +1,14 @@
+import joi from 'joi';
+import { idOrKeySchema } from './shared-schemas.js';
+import { DOCUMENT_REQUEST_TYPE, DAY_OF_WEEK } from '../constants.js';
+
+export const documentRequestDBSchema = joi.object({
+  _id: idOrKeySchema.required(),
+  documentId: idOrKeySchema.required(),
+  documentRevisionId: idOrKeySchema.required(),
+  type: joi.valid(...Object.values(DOCUMENT_REQUEST_TYPE)).required(),
+  loggedInUser: joi.boolean().required(),
+  createdOn: joi.date().required(),
+  createdOnDayOfWeek: joi.valid(...Object.values(DAY_OF_WEEK)).required()
```

Comment:
I have reconsidered some model field names for streamlining and clarity. Let me know if they make sense to you.

Reply:
They do make sense. For even more clarity we could rename the boolean flag even to `isUserLoggedIn`, what do you think?

Reply:
Even better, will change 👍